source/computer/commands.rst

```diff
@@ -63,54 +62,489 @@ Linux/macOS 下有成百上千个命令，每个命令都有众多选项。这
 
     -   其他
 
+        - `df`_
+        - `du`_
         - `ssh`_
         - `scp`_
         - `rsync`_
+        - `wget`_
+        - `top`_
 
 ----
 
 cat
 ---
 
+``cat`` 命令的命名来源于 con\ **cat**\ enate（即“拼接”）。该命令可以输出某个文件的
+内容，或将多个文件\ **拼接**\ 起来::
+
+    # 将一个文件的内容输出到终端
+    $ cat file
+    
+    # 同时显示行号
+    $ cat -n file
+    
+    # 将文件 file1 和 file2 合并到一个文件 target_file 中
+    $ cat file1 file2 > target_file
+
+diff
+----
+
+``diff`` 命令的命名来自 **diff**\ erence。该命令可以用来逐行比较文件的异同::
+
+    # 比较 /etc/passwd 和 /etc/passwd- 的异同
+    # 结果显示 /etc/passwd- 多了第 40 行
+    $ diff diff /etc/passwd /etc/passwd-
```

Comment:
```suggestion
    $ diff /etc/passwd /etc/passwd-
```

Reply:
I can't run this command on macOS.

Reply:
We may think about some other similar files.